buildconfig/pygame-stubs/__init__.pyi

```diff
@@ -1,14 +1,59 @@
-from typing import Any
-
-import pygame.event as event
-import pygame.joystick as joystick
-from pygame.rect import Rect as Rect
-from pygame.surface import *  # noqa: F403
+from typing import Any, Tuple, Callable, Union, Optional, overload, Type
 
+#Most useful stuff
+from pygame.constants import *
+import pygame.surface
+import pygame.rect
 import pygame.color
+import pygame.event
+import pygame.bufferproxy
+import pygame.draw
+import pygame.display
+import pygame.font
+import pygame.image
+import pygame.key
+import pygame.mixer
+import pygame.mouse
+import pygame.time
+import pygame.version
+
+# Advanced stuff
+import pygame.cursors
+import pygame.joystick
+import pygame.mask
+import pygame.sprite
+import pygame.transform
 import pygame.bufferproxy
+import pygame.pixelarray
+import pygame.pixelcopy
+import pygame.sndarray
+import pygame.surfarray
+import pygame.math
+# Other
+import pygame.scrap
+
+# This classes are auto imported with pygame, so I put their declaration here
+class Rect(pygame.rect.Rect): ...
```

Comment:
These aren't technically classes.
I wonder why this has been done, instead of the previous approach?
```python
Color = pygame.color.Color
```

Reply:
For example with Rect = pygame.rect.Rect if I use pygame.Rect((20, 20, 20, 20)) in my program PyCharm will give me warning saying parameters y, width, height unfilled. Meanwhile mypy will not find any warning (but there are a few otherproblems with PyCharm overloading) so I can rewrite them like they used to be